domain/src/main/java/org/oppia/android/domain/exploration/ExplorationProgressController.kt

```diff
@@ -720,4 +728,160 @@ class ExplorationProgressController @Inject constructor(
       lastPlayedTimestamp
     )
   }
+
```

Comment:
Kdocs missing

Reply:
done

Reply:
FWIW, KDocs aren't required for private members (per https://developer.android.com/kotlin/style-guide#usage), but they're definitely fine to add.